<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>132</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>648</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Progress</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="uiProgressGRP" >
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QProgressBar" name="uiSecondaryPBR" >
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>12</height>
         </size>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="textVisible" >
         <bool>false</bool>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QProgressBar" name="uiProgressPBR" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="textVisible" >
         <bool>false</bool>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="uiCaptionLBL" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Loading: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uiCancelBTN" >
          <property name="text" >
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
